src/gameplay/room.js

```diff
@@ -16,7 +16,7 @@ function Room(
     newRoomPassword_ = undefined;
   }
 
-  if (maxNumPlayers_ === '' || maxNumPlayers_ < 5 || maxNumPlayers_ > 10) {
+  if (!maxNumPlayers_.isInteger() || maxNumPlayers_ < 5 || maxNumPlayers_ > 10) {
```

Comment:
I'm not sure if `isInteger()` is a valid method on a string object. A better way might be to use `parseInt` and check for `NaN`. I'll do that now.